test/es_test_case.py

```diff
@@ -23,8 +23,10 @@ def setUpClass(cls):
         super().setUpClass()
         docker_client = docker.from_env()
         api_container_port = '9200/tcp'
+
         cls._es_docker_container = docker_client.containers.run("docker.elastic.co/elasticsearch/elasticsearch:5.5.3",
                                                                 detach=True,
+                                                                auto_remove=True, # Automatically remove an ES container upon stop/kill
```

Comment:
@hannes-ucsc For #159 and/or #172, could it be that there are not enough memory to run more than `n` containers of ElasticSearch? As my machine is way underpowered, I initially allocated 1GB of memory to Docker for Mac and then I constantly had issue of ElasticSearch silently failing to start. After I bump up the memory allocation to 2GB and extend the timeout to two minutes, then I have all tests pass.

Reply:
Possible. The most likely scenario is that it addresses #172 but you need Linux to reproduce. #159 is a long shot.

Since we can't conclusively answer the question as to whether that line addresses either or both of those issues, please turn this line into a separate commit with a title ending in ` (connected to #172)`.

Reply:
Understood.